src/ontology/cl-edit.owl

```diff
@@ -22773,6 +22775,24 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D000067170"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0017004 "telocyte")
 SubClassOf(obo:CL_0017004 ObjectIntersectionOf(obo:CL_0002320 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0120327)))
 
+# Class: obo:CL_0017010 (hillock cell of urethra)
```

Comment:
Consider adding the requested preferred label as a synonym. Consider adding additional synonyms requested in ticket if there is a dbxref to support.